packages/configuration-screen/src/components/ConfigurationCompletedStep.tsx

```diff
@@ -25,7 +25,7 @@ const ConfigurationCompletedStep: FC<{
             </h5>
             <div className="form-group">
                 <div className="button-group">
-                    <Button onClick={() => onBackClicked()} isSecondary>
+                    <Button onClick={(): void => onBackClicked()} isSecondary>
```

Comment:
`onBackClicked` sounds weird. Don't have to change it, but thought I'd mention it.

Reply:
You know what, I've never thought about that since I was working with french-speaking devs but that's a really good point